.gitignore

```diff
@@ -22,8 +22,8 @@ doc/build
 *.so
 
 # Ignore generated docs
-# */**/*.txt
-# !requirements.txt
+*/**/*.txt
```

Comment:
Not sure if this is working because there are some *.txt files in this PR... or maybe they are in a different place?